ui/test/services/bom_bar_client_test.dart

```diff
@@ -24,27 +24,45 @@ void main() {
     group('Projects', () {
       const projectId = 'projectId';
 
-      group('Get all projects', () {
-        test('queries projects', () async {
+      group('Get projects based on search query', () {
+        test('queries for all projects with no search fragment', () async {
           server.respondJson({
             'results': [
-              {'id': projectId}
+              {'id': projectId, 'title': 'ProjectA'},
```

Comment:
Why return more than one project? (Does that add value to the test?)

PS: All projects now have the same id, and are therefore actually the same project. :-)